src/Tasks/LegalProposition/Quiz.hs

```diff
@@ -43,7 +44,9 @@ generateLegalPropositionInst LegalPropositionConfig  {syntaxTreeConfig = SynTree
     return $ LegalPropositionInst
         { serialsOfWrong = fromList serialsOfWrong
         , pseudoFormulas = pseudoFormulas
+        , correctTrees = treeList
```

Comment:
Has this been tested? My hunch is that this is the list of all trees, even those of the answer options that were "made purposefully incorrect on output". But then that would mean that (if I read the implementation of the feedback for this task further above correctly) that more trees are output than reasonable. Only the "syntax trees of all well-formed formulas" among the answer options were intended to be printed.

Reply:
You are right. I fixed it.